src/docs.md

```diff
@@ -387,6 +387,17 @@ Sends a message to parent Cadenza window to close the window containing this app
 cadenzaClient.closeMe();
 ```
 
+### Listen on Cadenza closing Custom Application
+When closing custom application Cadenza doesn't send any event to it.
+To listen for event when window with custom application is closed, two standard events can be used:
+* `unload` - Fired when custom application is closed. Deprecated in documentation.
```

Comment:
Proposal: Link to the documenation on MDN instead of mentioning that it's deprecated.